.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Question: Could you explain what this does? I'm not sure if this was committed by accident.

Reply:
This is for using the repository in Github codespaces. It installs Rust and ninja to get started directly.
🙈 my poor mans way of developing on intel CPUs right now.